<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubscribePannel</class>
 <widget class="QWidget" name="SubscribePannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cb_version"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_match">
       <property name="text">
        <string>match</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Freq</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_freq">
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="cb_pkg"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_subscribe">
       <property name="text">
        <string>Subscribe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_remove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Resume</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
